src/ptvsd/adapter/__main__.py

```diff
@@ -57,6 +62,26 @@ def _parse_argv(argv):
         help="start the adapter in debugServer mode on the specified port",
     )
 
+    parser.add_argument(
+        "--host",
+        type=str,
+        nargs="?",
+        default="127.0.0.1",
+        const="127.0.0.1",
+        metavar="HOST",
+        help="start the adapter in debugServer mode on the specified host",
+    )
+
+    parser.add_argument(
+        "--connect-to-port",
+        type=int,
+        nargs="?",
+        default=None,
+        const=5678,
```

Comment:
Same thing here wrt `nargs` and `const`.

The switch also needs a better name, since the adapter can connect to a bunch of different things, and this one is for a very specific scenario. How about `--for-server-on-port`?